src/file-utils/edfi_lms_file_utils/file_reader.py

```diff
@@ -84,12 +117,10 @@ def get_all_system_activities(
     df = pd.concat(df_list)
     df.drop_duplicates(inplace=True)
 
-    return df
+    return pd.DataFrame(df)
```

Comment:
Surprisingly Pylance was complaining about this, as it inferred that `pd.concat` was returning something other than a DataFrame. I ran a little test in the Python interpreter and confirmed that `pd.DataFrame(df)` works just fine.